src/styles/styles.js

```diff
@@ -2356,6 +2359,42 @@ const styles = {
         ...whiteSpace.noWrap,
     },
 
+    offlineFeedback: {
+        deleted: {
+            textDecorationLine: 'line-through',
+            textDecorationStyle: 'solid',
+        },
+        pending: {
+            opacity: 0.5,
+        },
+        error: {
+            flexDirection: 'row',
+            justifyContent: 'center',
+            alignItems: 'center',
+        },
+        container: {
+            ...spacing.pv2,
+        },
+        text: {
+            color: themeColors.textSupporting,
+            flex: 1,
+            textAlignVertical: 'center',
+        },
+        close: {
+            marginTop: 'auto',
+            marginBottom: 'auto',
+        },
+        errorDot: {
+            borderColor: themeColors.textError,
+            borderRadius: 6,
+            borderWidth: 2,
```

Comment:
At least in web, these border properties seem to be unnecessary (except for borderRadius )

Reply:
I copied them from somewhere else... will do some tests in native and see if they make a difference there

Reply:
One thing with the error dot - I think we want to wrap it in a view that has 2px padding. This way the total bound of the red dot is 16x16, which will match other small icons we use for this kind of icon + message pattern. So essentially we'd get something like this:
<img width="687" alt="image" src="https://user-images.githubusercontent.com/2319350/179406100-4c3f7e04-f0df-4506-9783-f6e3c8e5e5de.png">


Let me know what you think about that. I know this isn't an "icon" per se, but all of our icons have a little bit of this white space around it. 

Actually, that being said, what we might want to do is just supply this as an .svg and then serve it up just like the rest of the icon system? Thoughts?

Reply:
But this would then become almost like an "exception" to the icon system, ya know?

Reply:
Ah ok, consistency then. I am fine with making it SVG, but have no idea how to do that, any tips?

Reply:
I got you! [dot-indicator.svg.zip](https://github.com/Expensify/App/files/9135935/dot-indicator.svg.zip)
 
So then just make sure this shows at 16x16, and uses red (danger) for the fill color.

Reply:
I believe so, you should see a bunch of other .svgs there